packages/pages/src/scaffold/modules/templates.ts

```diff
@@ -0,0 +1,85 @@
+export const moduleCode = (
+  moduleName: string,
+  useTailwind: boolean
+): string => {
+  const tailwind = useTailwind ? ` className="tailwind"` : ``;
+  const formattedModuleName =
+    moduleName.charAt(0).toUpperCase() + moduleName.slice(1);
+
+  return `import { Module, ModuleConfig, ModuleProps } from "@yext/pages/*";
+import { AnalyticsProvider } from "@yext/pages-components";
+import "./index.css";
+
+const templateData: ModuleProps = {
+  document: {
+    businessId: "REPLACE_ME",
+    siteId: "REPLACE_ME",
+    apiKey: "REPLACE_ME",
+    __: {
+      name: "${moduleName}",
+      staticPage: true
+    }
+  }, 
+  __meta: {
```

Comment:
I don't think this is used for anything.

Reply:
Hmmm I was getting some error w/o __meta, let me recreate

Reply:
Ah I get an error b/c `Property '__meta' is required in type 'TemplateProps<Record<string, any>>'` and `AnalyticsProviderProps.templateData: TemplateProps<Record<string, any>>`

Reply:
Ah ok. You can leave it then.